easybuild/easyconfigs/c/ccache/ccache-3.2.5.eb

```diff
@@ -0,0 +1,34 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+##
+easyblock = 'ConfigureMake'
+
+name = 'ccache'
+version = '3.2.5'
+
+homepage = 'http://ccache.samba.org/'
+description = """ccache-3.1.9: Cache for C/C++ compilers"""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['https://github.com/ccache/ccache/archive/']
+sources = ['v%(version)s.tar.gz']
+
+builddependencies = [('Autotools', '20150215')]
+dependencies = [
+    ('zlib', '1.2.8'),
```

Comment:
Sure, but a system without GCC is a lot less likely than a system without zlib...